src/core.js

```diff
@@ -150,7 +152,66 @@ export {
   Game
 }
 
+/**
+ * From http://stackoverflow.com/questions/6274339/how-can-i-shuffle-an-array-in-javascript
+ */
+function shuffle(array) {
+  let counter = array.length;
+
+  // While there are elements in the array
+  while (counter > 0) {
+    // Pick a random index
+    let index = Math.floor(Math.random() * counter);
+
+    // Decrease counter by 1
+    counter--;
+
+    // And swap the last element with it
+    let temp = array[counter];
+    array[counter] = array[index];
+    array[index] = temp;
+  }
+
+  return array;
+}
+
+function generateRandomCards() {
```

Comment:
Seems like a good function and helper functions to separate into a separate file.